src/bot/commands/ComboCommand.ts

```diff
@@ -18,13 +18,17 @@ export default class ComboCommand implements Command {
   }
 
   public run(message: Message, params: string[]) {
+    if (!message.member) {
+      return;
+    }
+
     if (params.length < this.NUMBER_OF_PARAMETERS) {
       message.channel.send(this.USAGE);
       return;
     }
 
-    const { voiceChannel } = message.member;
-    if (!voiceChannel) {
+    const { channel } = message.member.voice;
```

Comment:
One could also do things like `const { channel: voiceChannel } = message.member.voice;` to rename the destructured variable and not have to change subsequent usages.